### ISSUE TITLE
No documentation for java enum cases

### ISSUE THREAD

**Issue by liedQM** [Kotlin/dokka#476]
dokka 0.9.18 reports `No documentation for` for each case of a Java enum class even if this case is documented correctly.

Example:
```java
 /**
 * Enum representing the seven weekdays in German
 * (monday == first day of the week)
 */
public enum Weekdays {
    /**
     * The weekday case for `Monday`
     */
    MONDAY,

    /**
     * The weekday case for `Tuesday`
     */
    TUESDAY,
    /**
     * The weekday case for `Wednesday`
     */
    WEDNESDAY,
    /**
     * The weekday case for `Thursday`
     */
    THURSDAY,
    /**
     * The weekday case for `Friday`
     */
    FRIDAY,
    /**
     * The weekday case for `Saturday`
     */
    SATURDAY,
    /**
     * The weekday case for `Sunday`
     */
    SUNDAY
}
```

Error report:
```
> Task :dokka
No documentation for com.marcolied.examples.dokkajavaenumcase.Weekdays.MONDAY (Weekday.java:8)
No documentation for com.marcolied.examples.dokkajavaenumcase.Weekdays.TUESDAY (Weekday.java:13)
No documentation for com.marcolied.examples.dokkajavaenumcase.Weekdays.WEDNESDAY (Weekday.java:17)
No documentation for com.marcolied.examples.dokkajavaenumcase.Weekdays.THURSDAY (Weekday.java:21)
No documentation for com.marcolied.examples.dokkajavaenumcase.Weekdays.FRIDAY (Weekday.java:25)
No documentation for com.marcolied.examples.dokkajavaenumcase.Weekdays.SATURDAY (Weekday.java:29)
No documentation for com.marcolied.examples.dokkajavaenumcase.Weekdays.SUNDAY (Weekday.java:33)
```

I also uploaded a sample project to GitHub: https://github.com/MarcoLied/dokkaErrorJavaEnumCase

**Comment by masterjefferson** [comment]
We see the same problem.

**Comment by anupam-sapre** [comment]
I get the documentation with output type as javadoc but don't see it with output type html

**Comment by VSBochkov** [comment]
The same problem has been reproduced on 0.10.1

Could you please resolve it in prior
we are using djinni in project, generating types into java and would not like to have
Kotlin enum -> Java enum translation only because of absence enumeration cases documentation necessary for our library

Glad to see issue resolved in future release

**Comment by AfzalivE** [comment]
The reason for this seems that in `JavaLanguageService.kt`, `EnumItem` is a case in `renderClass` which only runs when the NodeKind is classLike but EnumItem is not in the classLike set.

So essentially, EnumItem node is never rendered for Java.

**Comment by MarcinAman** [comment]
I think the issue is fixed in 1.4.32
